src/tests/generators/account-generator.ts

```diff
@@ -0,0 +1,21 @@
+import { internet, name } from 'faker';
+
+interface SignUpInput {
```

Comment:
reuse object definitions, use the one from the previous PR once you separated it into a different file